src/connection.ts

```diff
@@ -394,6 +443,7 @@ export class Connection extends EventEmitter {
 
     // Return fulfillment and response packet
     const responsePacket = new Packet(requestPacket.sequence, IlpPacketType.Fulfill, prepare.amount, responseFrames)
+    this._totalReceived = this._totalReceived.plus(responsePacket.prepareAmount)
```

Comment:
nitpick: seems a little roundabout to get the prepare amount from the response packet instead of just `prepare.amount`

Reply:
Agreed, moved it to prepare.amount